Makefile

```diff
@@ -12,52 +12,91 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+# Deliverables path
 DESTDIR ?= /usr/local
-
-VERSION=$(shell git describe --match 'v[0-9]*' --dirty='.m' --always --tags)
+BINDIR ?= $(DESTDIR)/bin
+
+# Tools path
+RM ?= /bin/rm
+ECHO ?= /bin/echo
+DOCKER ?= /bin/docker
+GO ?= /bin/go
+MKDIR ?= /bin/mkdir
+TAR ?= /bin/tar
+INSTALL ?= /bin/install
+GIT ?= /bin/git
```

Comment:
docker, go, git are very unlikely to be installed under `/bin`.

Why not just use the default binaries in the PATH?

Reply:
How about using `GIT ?= $(which git)`

Reply:
Isn't `GIT ?= git` enough?

Reply:
Yeah that should suffice, I though of using absolute paths. A day might come where I have multiple versions of git or docker installed in my machine and I want to use given version of docker or git for testing. But `GIT ?= git` should suffice.

I though something like this:
```
# Tools path
ECHO ?= $(shell which echo)
DOCKER ?= $(shell which docker)
GO ?= $(shell which go)
MKDIR ?= $(shell which mkdir)
TAR ?= $(shell which tar)
INSTALL ?= $(shell which install)
GIT ?= $(shell which git)
```

Reply:
`which` is often missing on recent distros

Reply:
Yeah, I am also thinking same. Let's go with your approach. Thanks.